contrib/win32/win32compat/console.c

```diff
@@ -1091,13 +1094,20 @@ ConMoveVisibleWindow(int offset)
 				ConScrollDown(0, consoleInfo.dwSize.Y - 1);
 
 			if (GetConsoleScreenBufferInfo(hOutputConsole, &consoleInfo))
-				memcpy(&visibleWindowRect, &consoleInfo.srWindow, sizeof(visibleWindowRect));
+				if (memcpy_s(&visibleWindowRect, sizeof(visibleWindowRect), &consoleInfo.srWindow, sizeof(visibleWindowRect))) {					
+					error("memcpy_s failed: %d.", errno);
+					return;
+				}
+
```

Comment:
This else now refers to the if (memcpy_s) loop above but it has to be the if (GetConsoleScreenBufferInfo)